ctest-test/build.rs

```diff
@@ -1,107 +1,62 @@
-use std::process::Command;
```

Comment:
Could `ctest_next` be added to the tests without removing `ctest`? Or do the updates to tests make something fail?

Reply:
Can't you use the skips?

Reply:
Ah, I guess `skip_function` and `skip_static` haven't been added yet. If you add them then you should be able to `cfg.skip_function(|_f| true); cfg.skip_static(|_s| true)` in the tests until it works, without blocking on everything else.

The same is true for `libc-test`